### PR TITLE
Fix EZP-25159: Implement missing limitation mappers Node and Subtree

### PR DESCRIPTION
**Pull request by glye**
> Implements https://jira.ez.no/browse/EZP-25159
> State: Ready for review
> Also requires https://github.com/ezsystems/PlatformUIBundle/pull/433

Adds the missing, UDW-based limitation mappers, used when editing role policies. Extracted from https://github.com/ezsystems/repository-forms/pull/58

- [x] Node
- [x] Subtree

### CODE REVIEW

**Comment by lolautruche** [comment]
Mmm why do you want to merge into `ezp-25021_missing_limitation_mappers` and not into `master`?

File: bundle/Resources/config/role.yml [hunk 1]
```diff
@@ -177,3 +180,18 @@ services:
...
+        calls:
+            - [setFormTemplate, [%ezrepoforms.limitation.udw.template%]]
+
+    ezrepoforms.limitation.form_mapper.subtree:
+        class: %ezrepoforms.limitation.form_mapper.subtree.class%
+        arguments: [@ezpublish.api.service.location]
+        tags:
+            - { name: ez.limitation.formMapper, limitationType: Subtree }
+        calls:
+            - [setFormTemplate, [%ezrepoforms.limitation.udw.template%]]
```
Comment (lolautruche) [inline bundle/Resources/config/role.yml]: As already suggested (IIRC), this call and the same one for the location mapper can be abstracted in an abstract service.

**Comment by glye** [comment]
@lolautruche Only temporarily, until `ezp-25021_missing_limitation_mappers` is merged into master. I see it is, now.

**Comment by glye** [comment]
Closing in favour of https://github.com/ezsystems/repository-forms/pull/64

**Pull request closed without merging**